x-pack/plugins/observability/public/components/app/observability_status/observability_status_box.tsx

```diff
@@ -81,34 +94,41 @@ export function CompletedStatusBox({
           </EuiFlexItem>
         ))}
       </EuiFlexGroup>
+
+      <EuiFlexGroup>
+        <EuiFlexItem>
+          <EuiButton color="primary" size="s" href={goToAppLink}>
+            {goToAppTitle}
+          </EuiButton>
+        </EuiFlexItem>
+      </EuiFlexGroup>
     </EuiPanel>
   );
 }
 
 export function EmptyStatusBox({
-  dataSourceName,
+  title,
   description,
   learnMoreLink,
+  addTitle,
+  addLink,
 }: ObservabilityStatusBoxProps) {
   return (
-    <EuiPanel color="subdued" style={{ marginBottom: 20 }}>
+    <EuiPanel color="warning" hasBorder={true} style={{ marginBottom: 20 }}>
```

Comment:
```suggestion
    <EuiPanel color="warning" hasBorder={true}>
```
Not sure why there was an added `20px` margin between the status boxes in the "next steps" area.

<img width="453" alt="CleanShot 2022-03-21 at 13 04 48@2x" src="https://user-images.githubusercontent.com/4104278/159257505-f6c50d8b-5568-4189-b791-23ee258317ff.png">

vs. 

<img width="463" alt="CleanShot 2022-03-21 at 13 05 17@2x" src="https://user-images.githubusercontent.com/4104278/159257554-940539b4-56ac-4bfd-aa43-a3920239ec71.png">